scripts/install_bmi.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -ex
+sudo apt-get install -y python-pip
+### Packages for BMI
+sudo apt-get install -y tgt sqlite3 virtinst
+
+git clone http://github.com/cci-moc/hil
+
+pushd hil/
+sudo apt-get install -y libvirt-bin bridge-utils ipmitool telnet apache2 libapache2-mod-wsgi python-pip qemu-kvm python-libvirt python-psycopg2 vlan net-tools icu-devtools libicu-dev libxml2-dev libxslt1-dev python-virtualenv python3-virtualenv virtualenv # For Ubuntu
+
+virtualenv .venv
+source .venv/bin/activate
+pip install -e .
+
+cp examples/haas.cfg.dev-no-hardware ./haas.cfg
+
+export HAAS_ENDPOINT=http://127.0.0.1:5000
+export HAAS_USERNAME=haas
+export HAAS_PASSWORD=secret
+
+# Start Haas services
+haas-admin db create
+haas serve 5000&
+haas serve_networks&
```

Comment:
Just followed the docs here: https://github.com/CCI-MOC/hil/blob/master/docs/INSTALL-devel.rst#start-the-server

I could just remove the line if you think it's of no use.